core/commands/dag/dag.go

```diff
@@ -33,6 +39,7 @@ to deprecate and replace the existing 'ipfs object' command moving forward.
 		"put":     DagPutCmd,
 		"get":     DagGetCmd,
 		"resolve": DagResolveCmd,
+		"export":  DagExportCmd,
```

Comment:
this file is quickly reaching the level of complexity where it may make sense to split the different sub-commands into their own files.

Reply:
@Stebalien thoughts?

Reply:
Yeah, that's a fair point. But I'd leave it this way then break it up in a followup patch to reduce the diff.